hotdoc/core/symbols.py

```diff
@@ -28,6 +28,8 @@ class Symbol(Base):
     display_name = Column(String)
     filename = Column(String)
     lineno = Column(Integer)
+    extent_start = Column(Integer)
```

Comment:
Hm I'm not *completely* happy with this, as it kind of overlaps with lineno, however I'll merge that for now and refactor later, probably by having "source_start" and "source_end" be a "PickleType" kind of column, which will pickle a "Location" containing both line number and column, which the c extension knows about too if I'm not mistaken.